app/controllers/internal/UpscanCallbackController.scala

```diff
@@ -45,13 +45,10 @@ class UpscanCallbackController @Inject()(repository: UploadJourneyRepository)
               seqOfChecksums => {
                 if(seqOfChecksums.contains(callbackModel.uploadDetails.get.checksum)) {
                   logger.debug("[UpscanCallbackController][callbackFromUpscan] - Checksum is already in Mongo. Marking file as duplicate.")
-                  val duplicateCallbackModel = callbackModel.copy(
-                    fileStatus = UploadStatusEnum.FAILED,
-                    failureDetails = Some(FailureDetails(FailureReasonEnum.DUPLICATE, "upscan.duplicateFile")),
```

Comment:
There is a case statement for getting the localised message, you may want to remove it now as it's not needed:
https://github.com/hmrc/penalties-appeals-frontend/blob/0fd60ff0937b953b9fa5331d8461fa14c75baecb/app/helpers/UpscanMessageHelper.scala#L29